db/schema.rb

```diff
@@ -10,12 +10,41 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_11_18_130234) do
+ActiveRecord::Schema.define(version: 2019_11_18_150004) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
+  create_table "activities", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "identifier"
+    t.string "sector"
+    t.string "title"
+    t.text "description"
+    t.string "status"
+    t.date "planned_start_date"
+    t.date "planned_end_date"
+    t.date "actual_start_date"
+    t.date "actual_end_date"
+    t.string "recipient_region"
+    t.string "flow"
+    t.string "finance"
+    t.string "aid_type"
+    t.string "tied_status"
+    t.uuid "fund_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["fund_id"], name: "index_activities_on_fund_id"
```

Comment:
On reflection I think that we should use a polymorphic association here to relate an activity to anyone of our 4 known parent types, rather than fund specifically. It seems to be the right tool for what we want to achieve here.

```
activity.hierachy => fund
activity.hierachy => programme
activity.hierachy => project
activity.hierachy => grant
```

I'm not sure if `hierarchy` is a good name but it is loosely related due to IATI terms. Is there a better one?

Hopefully by using a polymorphic association we will make these associations easier to manage, especially helping us if we need to add, rename or remove a given type.

Reply:
The only downside of this is I can't seem to spell `hierarchy` consistently 😆